examples/reference/models/Dash.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np

```

Comment:
Just confirming, do we intend to have python files with upper-case characters -- **D**ask.py?

Reply:
It's consistent with the rest of the directory. Usually I would advise not to use uppercase module names. But these files are standalone and not part of  package.

The names describe class names, so I prefer the filenames to match the exact classname here.

Reply:
Yes in this specific instance these are essentially test files an the file names refer to corresponding class names